subsys/bluetooth/controller/ll_sw/lll.h

```diff
@@ -39,6 +39,11 @@
 #endif /* !CONFIG_BT_CTLR_ADV_EXT */
 #endif /* CONFIG_BT_OBSERVER */
 
+#if defined(CONFIG_BT_CTLR_ADV_EXT)
```

Comment:
if you return status, you will not need these?

Reply:
Removed